File: trunk/src/com/ff/ui/form/Field.java
package com.ff.ui.form;

import com.ff.Component;
import com.ff.annotation.Property;

public class Field extends Component{

	@Property
	private String name;
	@Property
	private String value;

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getValue() {
		return value;
	}

	public void setValue(String value) {
		this.value = value;
	}
	
	
	
}

File: zkext/trunk/src/web/js/zkext/ui/form/Field.js
/* Field.js*/

zkext.ui.form.Field = zk.$extends(zkext.ui.Component,{	 
	$define: {	 
		name:function(val){
			 val = val.replace(".",ESCAPE_FIELD);
			 this._name = val; 
			 this.setProperty("name",null,val);
		},		
		value:function(val){
			this.setProperty("value","setValue",val+"");
		},
		label:function(val){
			this.setProperty("fieldLabel","setLabel",val);
			//this.setProperty("boxLabel","setLabel",val);
		},
		boxLabel:function(val){
			this.setProperty("boxLabel","setBoxLabel",val);
		}		
	},
	
	getModel:function(){
		return this.ext_.getValue();
	}
});